.github/workflows/validate_and_publish_release.yml

```diff
@@ -97,3 +97,10 @@ jobs:
           github_token: ${{ secrets.WRITE_ACCESS_TOKEN }}
           branch: latest-snapshot
           force: true
+
+      # Publish an updated release index page to the website
+      - name: Invoke workflow to publish website
+        uses: benc-uk/workflow-dispatch@4c044c1613fabbe5250deadc65452d54c4ad4fc7 # v1
+        with:
+          workflow: Publish Website
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
```

Comment:
Another option would be to use the 'release' trigger: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release in here: https://github.com/iree-org/iree/blob/10a3b04950528af67dfba8f9ac125be37959c597/.github/workflows/publish_website.yml#L8-L14

Reply:
Yeah, I like that more 
```yaml
on:
  release:
    types: [prereleased, released]
```

Reply:
Oh, or just `published`... and maybe `deleted` (if we yank a release)

Reply:
Oooh I like the release trigger idea. That makes the workflow relying on the release the one that has to know about what it's doing rather than having the release have to know about everything that depends on it

Reply:
Also nice because it means that if someone downstream workflow fails it doesn't fail the release workflow